bin/nexus-blobstores-create

```diff
@@ -64,12 +64,12 @@ while [[ $# -gt 0 ]]; do
             -d @-
         status=$?
         if [[ $status -eq 0 ]]; then
-            echo >&2 "INFO Successfully created ${blobstore_type} blobstore: ${blobstore_name}"
+            echo >&2 " OK created"
```

Comment:
IUF will not be able to recognize this message if INFO is removed.

Reply:
It is better if each message can stand alone with all the context.  That is to say, the original message is better because it is more complete, the new message doesn't say _what_ was created.